src/Tribe/PUE/Checker.php

```diff
@@ -488,45 +488,25 @@ private function get_api_message( $info ) {
 		}
 
 		/**
-		 * Echo JSON formatted errors
+		 * Displays an error notice if a premium plugin is activated but no license is entered
+		 *
+		 * @since 4.3
 		 */
 		public function display_json_error() {
-			$pluginInfo       = $this->json_error;
-			$update_dismissed = $this->get_option( $this->dismiss_upgrade );
-
-			$is_dismissed = ! empty( $update_dismissed ) && in_array( $pluginInfo->version, $update_dismissed ) ? true : false;
-
-			if ( $is_dismissed ) {
-				return;
-			}
+			$plugin_info = (array) $this->json_error;
 
 			if ( ! current_user_can( 'administrator' ) ) {
 				return;
 			}
 
-			//only display messages if there is a new version of the plugin.
-			if ( version_compare( $pluginInfo->version, $this->get_installed_version(), '>' ) ) {
-				if ( empty( $pluginInfo->api_invalid ) || $pluginInfo->api_invalid != 1 ) {
-					return;
-				}
+			if ( '' != $this->get_plugin_name() ) {
 
-				$msg = $this->get_api_message( $pluginInfo );
-
-				//Dismiss code idea below is obtained from the Gravity Forms Plugin by rocketgenius.com
+				$msg     = $plugin_info['api_invalid_message'];
```

Comment:
Maybe I'm reading this wrong, but it seems like we are losing the error message provided by `get_api_message()` that was in this method beforehand. Is that intended?